Makefile

```diff
@@ -24,7 +24,8 @@ build-all:
 	gox -verbose \
 	-ldflags "-X main.version=${VERSION}" \
 	-os="linux darwin windows " \
-	-arch="amd64 386" \
+	-arch="amd64 386 armv5 armv6 armv7 arm64" \
```

Comment:
I just tried it out and I don't think armv{5,6,7} are valid. It just ignores the arch's it doesn't know about.

I believe only `arm` and `arm64` are valid.